### PR TITLE
added a command to install package with packages managers ( only support portage )

### PR DESCRIPTION
**Pull request by marius851000**
add the command package, in which you can specify packages for package manager.

example : 
- package : {
  portage : games-rpg/freedroid
}

### CODE REVIEW

File: lutris/installer/commands.py [hunk 1]
```diff
@@ -292,6 +292,26 @@ def move(self, params):
...
+        # get the package id
+        package = params.get(packageManagerToUse)
+        if type(package) != str:
+            raise ScriptingError("no package specified for the '"+packageManagerToUse+"' package manager")
+        # install via the package manager
+        if packageManagerToUse == "portage":# for gentoo/linux, using emerge
+            return self.execute("kdesu -t -c 'emerge "+package+" --nospinner'")
+        # TODO: add the apt package manager
+        else:
+            raise ScriptingError("Lutris isn't compatible with the '"+packageManager+"' package manager at this time.")
```
Comment (strycore) [inline lutris/installer/commands.py]: string interpolation

File: lutris/installer/commands.py [hunk 2]
```diff
@@ -292,6 +292,26 @@ def move(self, params):
             # Change game file reference so it can be used as executable
             self.game_files['src'] = src
 
+    def package(self, params):
+        """install a package on the preffered package manager"""
+        import os.path
+        # get the sytem package manager
+        portageInstalled = os.path.isdir("/usr/share/portage")
+        if portageInstalled:
+            packageManagerToUse = "portage"
```
Comment (strycore) [inline lutris/installer/commands.py]: naming

File: lutris/installer/commands.py [hunk 3]
```diff
@@ -292,6 +292,26 @@ def move(self, params):
             # Change game file reference so it can be used as executable
             self.game_files['src'] = src
 
+    def package(self, params):
+        """install a package on the preffered package manager"""
+        import os.path
```
Comment (strycore) [inline lutris/installer/commands.py]: imports at the top

File: lutris/installer/commands.py [hunk 4]
```diff
@@ -292,6 +292,26 @@ def move(self, params):
...
+            packageManagerToUse = "portage"
+        else:
+            raise ScriptingError("No compatible system manager found on your system. actually, only portage work.")
+        # get the package id
+        package = params.get(packageManagerToUse)
+        if type(package) != str:
+            raise ScriptingError("no package specified for the '"+packageManagerToUse+"' package manager")
+        # install via the package manager
+        if packageManagerToUse == "portage":# for gentoo/linux, using emerge
+            return self.execute("kdesu -t -c 'emerge "+package+" --nospinner'")
```
Comment (strycore) [inline lutris/installer/commands.py]: don't use kdesu.

Comment (marius851000) [reply lutris/installer/commands.py]: I didn't know by what I can replace this. I've solved the other issue in my local version.

**Pull request closed without merging**